docs/roman/gain_scale/description.rst

```diff
@@ -0,0 +1,37 @@
+Description
+============
+
+The ``gain_scale`` step rescales pixel values in roman countrate
+science data products in order to correct for the effect of using
+a non-standard detector gain setting. The countrate data are
+rescaled to make them appear as if they had been obtained using
+the standard gain setting.
+
+The `gain_scale` step is applied at the end of the
+TBD 
+#:ref:`calwebb_detector1 <calwebb_detector1>` 
+pipeline, after the :ref:`ramp_fit <ramp_fitting_step>` step has been applied. It is applied to both the "rate" and "rateints" products from
+:ref:`ramp_fit <ramp_fitting_step>`, if both
+types of products were created. The science (SCI) and error (ERR)
+arrays are multiplied by the gain factor, and the Poisson
+variance (VAR_POISSON) and read noise variance (VAR_RNOISE) arrays
+are multiplied by the square of the gain factor.
+
+The scaling factor is obtained from the "gain_factor" keyword in the
```

Comment:
keyword --> attribute  or value?

Reply:
Updated these to keyword attributes. Blindly replacing keyword with attribute is not advisable, since the metadata contains keyword attributes, library attributes, catalogue attributes.. attributes are every entry with a colon in the metadata.